nautobot/extras/views.py

```diff
@@ -1229,32 +1229,16 @@ def get_extra_context(self, request, instance):
 #
 
 
-class GraphQLQueryListView(generic.ObjectListView):
+class GraphQLQueryUiViewSet(NautobotUIViewSet):
```

Comment:
1. Should be `UIViewSet` not `UiViewSet`
2. Since we don't support bulk-edit for this view, we can't use the full `NautobotUIViewSet` as base class, need to build it up from the individual mixins instead, i.e:

```suggestion
class GraphQLQueryUIViewSet(
    mixins.ObjectDetailViewMixin,
    mixins.ObjectListViewMixin,
    mixins.ObjectEditViewMixin,
    mixins.ObjectDestroyViewMixin,
    mixins.ObjectBulkDestroyViewMixin,
    mixins.ObjectChangeLogViewMixin,
    mixins.ObjectNotesViewMixin,
):
```